pandas/tests/extension/base/__init__.py

```diff
@@ -31,6 +31,14 @@ class TestMyDtype(BaseDtypeTests):
 Your class ``TestDtype`` will inherit all the tests defined on
 ``BaseDtypeTests``. pytest's fixture discover will supply your ``dtype``
 wherever the test requires it. You're free to implement additional tests.
+
+All the tests in these modules use ``self.assert_frame_equal`` or
+``self.assert_series_equal`` for dataframe or series comparisons. By default,
+they use the usual ``pandas.util.testing.assert_frame_equal`` and
```

Comment:
pandas.util.testing.assert_frame_equal -> pandas.testing.assert_frame_equal (that is the public API path)